unstructured/partition/pdf.py

```diff
@@ -572,7 +572,7 @@ def _add_pytesseract_bboxes_to_elements(
     boxes = bboxes_string.strip().split("\n")
     box_idx = 0
     for element in elements:
-        if not element.text:
```

Comment:
What logic are we looking for here, because it looks like anything truthy in `boxes[box_idx][0]` will cause it to skip this element, and that doesn't sound right to me.